helm/templates/statefulset.yaml

```diff
@@ -32,12 +32,10 @@ spec:
       - name: immudb-storage
         persistentVolumeClaim:
           claimName: {{ include "immudb.fullname" . }}
-      securityContext:
```

Comment:
This is the PodSecurityContext, I believe this should be kept to apply these settings to all containers in the pod. The security context can then be applied at each container level.
```
securityContext:
    {{- toYaml .Values.podSecurityContext | nindent 8 }}
containers:
    - name: {{ .Chart.Name }}
       securityContext: {{- toYaml .Values.securityContext | nindent 12 }}
```

Reply:
cc: @byo @SimoneLazzaris

Reply:
The PodSecurityContext should indeed remain and it is also set in lines 29,30.  The `securityContext` key currently appears twice in the spec section, therefore  one occurrence has been removed in PR.

Reply:
@SimoneLazzaris could you have a look at this? cc: @byo